example/android/app/build.gradle

```diff
@@ -143,30 +143,28 @@ android {
         versionName "1.0"
         buildConfigField "boolean", "IS_NEW_ARCHITECTURE_ENABLED", isNewArchitectureEnabled().toString()
         if (isNewArchitectureEnabled()) {
-            // We configure the NDK build only if you decide to opt-in for the New Architecture.
+            // We configure the CMake build only if you decide to opt-in for the New Architecture.
```

Comment:
You didn't add this import to this file: `import org.apache.tools.ant.taskdefs.condition.Os`

Reply:
The `Os` is not used anywhere. IIRC it was used to check if the host OS is Windows, but the check has been removed since.